lib/src/linking.rs

```diff
@@ -115,10 +115,17 @@ pub(crate) fn create_store(
 fn make_wasi_ctx(ctx: &ExecuteCtx, session: &Session) -> Result<WasiCtx, anyhow::Error> {
     let mut wasi_ctx = WasiCtxBuilder::new();
 
-    // Viceroy provides a subset of the `FASTLY_*` environment variables that the production
+    // Viceroy provides the same `FASTLY_*` environment variables that the production
```

Comment:
Would you mind clarifying here that values will be defaulted to stub values when they can't be inherited?

Reply:
Oh, but I don't think they are inherited, are they? if you mean from the host machine's environment

Reply:
You're totally right -- I misread this as being lookups in the env. Ignore that original request :sweat_smile: